src/main/views/webContentEvents.ts

```diff
@@ -352,6 +358,12 @@ export class WebContentsEventManager {
         const newWindow = this.generateNewWindowListener(contents.id, spellcheck);
         contents.setWindowOpenHandler(newWindow);
 
+        // Defer handling of new popup windows to PluginsPopUpsManager. These still need to be
+        // previously allowed from generateNewWindowListener through PluginsPopUpsManager.handleNewWindow.
+        contents.on('did-create-window', (win: BrowserWindow, details: Electron.DidCreateWindowDetails) => {
+            PluginsPopUpsManager.handleCreateWindow(win, details);
```

Comment:
Won't this create two windows using the existing allowance for plugin popups? See https://github.com/mattermost/desktop/pull/3136/files#diff-daeb8475a075c7e3efcef7ebc2b325e61c5275b03fdc4911c4d48e6b064bcfa5L233

Reply:
It shouldn't interfere since the new manager only deals with `about:blank` targets at the moment. Eventually, I'd like to unify both of these popup handlers.

Reply:
Test it for me if you would please, if it works then yeah this is fine for now.

Reply:
Yeah, I gave it a try to confirm I wasn't completely wrong :p